war/src/main/js/add-item.js

```diff
@@ -107,7 +107,6 @@ $.when(getItems()).done(function(data) {
           btn.removeClass('disabled');
           btn.prop('disabled', false);
         }
-        btn.focus();
```

Comment:
If the GET request sent after each character and all three checks pass (js allows to create a job) then the enableSubmit function is called and OK button is focused after each character typed.
Kirill suggested not to remove this focus and create different functions for blur and keyup events. So this is not finished also.